Sources/HipMobile.Tests/ServiceAccessLayer/ContentApiAccesses/FeatureToggleApiAccessTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (C) 2017 History in Paderborn App - Universität Paderborn
+//  
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//  
+//      http://www.apache.org/licenses/LICENSE-2.0
+//  
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NSubstitute;
+using NUnit.Framework;
+using PaderbornUniversity.SILab.Hip.Mobile.Shared.ServiceAccessLayer;
+using PaderbornUniversity.SILab.Hip.Mobile.Shared.ServiceAccessLayer.FeatureToggleApiDto;
+
+namespace PaderbornUniversity.SILab.Hip.Mobile.HipMobileTests.ServiceAccessLayer.ContentApiAccesses
+{
+    [TestFixture]
+    public class FeatureToggleApiAccessTest
+    {
+        private readonly IContentApiClient contentApiSubstitute;
+        private readonly FeatureToggleApiAccess client;
+
+        public FeatureToggleApiAccessTest()
+        {
+            contentApiSubstitute = Substitute.For<IContentApiClient>();
+            client = new FeatureToggleApiAccess(contentApiSubstitute);
+        }
+
+        [Test, Category("UnitTest")]
+        public async Task GetEnabledFeaturesTest()
+        {
+            var json = @"
+[
```

Comment:
Is this the way the test-json files have to be indented or can they be more aligned with the rest of the method?